packages/engine/src/behave-graph/nodes/Profiles/Scene/readSceneGraphs.test.ts

```diff
@@ -0,0 +1,84 @@
+/*
+CPAL-1.0 License
+
```

Comment:
We can get rid of all the behave graph / behave flow tests, it's additional bulk we don't need